transactions/lockedTokens/user/get_total_balance.cdc

```diff
@@ -54,12 +52,12 @@ access(all) fun main(address: Address): UFix64 {
     }
 
     // Get the locked account public capability
-    let lockedAccountInfoCap = account
-        .getCapability<&LockedTokens.TokenHolder>(
+    let optionalLockedAccountInfoRef = account
+        .capabilities.get<&LockedTokens.TokenHolder>(
```

Comment:
Should this be a borrow? The name and the usage below indicates it's used like a reference instead of a capability.

```suggestion
        .capabilities.borrow<&LockedTokens.TokenHolder>(
```

Reply:
Updated [67ea336](https://github.com/onflow/flow-core-contracts/pull/382/commits/67ea336c5fd8d8f8bfcc65ba822fee40958eb7b6)